File: CS/03_Slides/Layout/AddDifferentLayoutSlides.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Presentation;

namespace AddDifferentLayoutSlides
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();            
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            //Create a PPT document
            Presentation presentation = new Presentation();

            //Remove the default slide
            presentation.Slides.RemoveAt(0);

            //Loop through slide layouts
            foreach (SlideLayoutType type in Enum.GetValues(typeof(SlideLayoutType)))
            {
                //Append slide by specifing slide layout
                presentation.Slides.Append(type);
            }

            //Save the document
            string result = "AddDifferentLayoutSlides_result.pptx";
            presentation.SaveToFile(result, FileFormat.Pptx2013);

            //Launch the file
            OutputViewer(result);
        }
        private void OutputViewer(string filename)
        {
            try
            {
                System.Diagnostics.Process.Start(filename);
            }
            catch { }
        }
    }
}

File: VB/03_Slides/AddDifferentLayoutSlides.vb
Imports System.ComponentModel
Imports System.Text
Imports Spire.Presentation

Namespace AddDifferentLayoutSlides
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			'Create a PPT document
			Dim presentation As New Presentation()

			'Remove the default slide
			presentation.Slides.RemoveAt(0)

			'Loop through slide layouts
			For Each type As SlideLayoutType In System.Enum.GetValues(GetType(SlideLayoutType))
				'Append slide by specifing slide layout
				presentation.Slides.Append(type)
			Next type

			'Save the document
			Dim result As String = "AddDifferentLayoutSlides_result.pptx"
			presentation.SaveToFile(result, FileFormat.Pptx2013)

			'Launch the file
			OutputViewer(result)
		End Sub
		Private Sub OutputViewer(ByVal filename As String)
			Try
				Process.Start(filename)
			Catch
			End Try
		End Sub
	End Class
End Namespace